assets/src/edit-story/components/panels/layer/useElements.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../../app';
+
+import IconBackground from './layer_bg.svg';
+import IconMedia from './layer_media.svg';
+import IconVideo from './layer_video.svg';
+import IconText from './layer_text.svg';
+
+function getIconForElementType( type ) {
+	switch ( type ) {
+		case 'video':
+			return <IconVideo />;
```

Comment:
Ditto. Do it via `getDefinitionForType`?

Reply:
Fixed with other stuff in 07f0e2a.